draft-ietf-dhc-dhcpv4-over-dhcpv6-ra.md

```diff
@@ -245,7 +247,7 @@ information in the encapsulated message.
 As shown in {{fig_4o6RA_RA}}, the introduction of 4o6 at the
 edge of the IPv6 network hides the L2 network from the DHCPv6 RA.
 
-In order to preserve the topology information, it is recommended that the
+In order to preserve the topology information, it is RECOMMENDED that the
```

Comment:
I'm not sure if this needs normative language as this is really just some advise and not something needed for the protocol to interoperate.

Reply:
I would keep upcase, we actually want that the topology information is preserved.
We may have written in a different way, though. 
Let see if this is accepted.